include/zephyr/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -192,6 +192,9 @@ SECTIONS
 
 #include <zephyr/linker/kobject-rom.ld>
 
+	*(.shared_isr_args)
+	*(.shared_isr)
```

Comment:
Is this going to work with `CONFIG_CMAKE_LINKER_GENERATOR=y` ?

Reply:
I've been testing with the cdc_acm_composite sample and enabling CONFIG_CMAKE_LINKER_GENERATOR, without any of my changes, generates build errors.

But setting CONFIG_CMAKE_LINKER_GENERATOR=y would likely fail with my changes too, so the generated linker file will need to include the .shared_isr and .shared_isr_args sections.